code/modules/events/sentience.dm

```diff
@@ -4,7 +4,7 @@
 	processing = FALSE //so it won't fire again in next tick
 
 	var/ghostmsg = "Do you want to awaken as a sentient being?"
-	var/list/candidates = pollCandidates(ghostmsg, ROLE_SENTIENT, 1)
+	var/list/candidates = SSghost_spawns.poll_candidates(ghostmsg, ROLE_SENTIENT, TRUE)
```

Comment:
Is `ghostmsg` only used on this line? If so, remove the var entirely and substitute the text in

Reply:
Donezo